src/fvm_multicell.hpp

```diff
@@ -465,11 +447,13 @@ void fvm_multicell<Backend>::initialize(
 
     // Allocate scratch storage for calculating quantities used to build the
     // linear system: these will later be copied into target-specific storage
-    // as need be.
-    // Initialize to zero, because the results therin are calculated via accumulation.
-    std::vector<value_type> tmp_face_conductance(ncomp, 0.);
-    std::vector<value_type> tmp_cv_areas(ncomp, 0.);
-    std::vector<value_type> tmp_cv_capacitance(ncomp, 0.);
+
+    // face_conductance_[i] = area_face  / (r_L * delta_x);
```

Comment:
This was a cut and paste of the comment that was above the variable definition before the refactoring... stale comments!